integrations/anthropic/src/index.ts

```diff
@@ -41,13 +41,13 @@ export default new bp.Integration({
   actions: {
     generateContent: async ({ input, logger }) => {
       return await generateContent<ModelId>(<llm.GenerateContentInput>input, anthropic, logger, {
-        models,
+        models: languageModels,
         defaultModel: 'claude-3-5-sonnet-20240620',
       })
     },
-    listModels: async ({}) => {
+    listLanguageModels: async ({}) => {
```

Comment:
**todo:** this is a breaking change, so don't forget to bump anthropic's version. If it's already public -> bump by a major, else bump by whatever you want